cmd/run.go

```diff
@@ -770,3 +804,53 @@ func createRollupDataQuerier(ctx context.Context,
 			return polygonrollupmanager.NewPolygonrollupmanager(rollupAddr, client)
 		})
 }
+
+// runTxnSenderBackfill runs the txn_sender backfilling process
+func runTxnSenderBackfill(
+	ctx context.Context,
+	cfg bridgesync.Config,
+	client aggkittypes.EthClienter,
+	wg *sync.WaitGroup,
+) error {
+	const backfillTimeoutMinutes = 10
+
+	// Only run backfilling if we have a database path configured
+	if cfg.DBPath == "" {
+		log.Debug("No database path configured, skipping txn_sender backfilling")
+		return nil
+	}
+
+	// Defer WaitGroup Done to ensure cleanup on exit
+	defer wg.Done()
+
+	log.Info("Starting txn_sender backfilling process")
+
+	// Create backfill instance
+	backfiller, err := bridgesync.NewBackfillTxnSender(
+		cfg.DBPath,
+		client,
+		cfg.BridgeAddr,
+		log.WithFields("module", "tx-sender-backfill"),
+	)
+	if err != nil {
+		return fmt.Errorf("failed to create backfill instance: %w", err)
+	}
+	defer backfiller.Close()
```

Comment:
We might consider to call the `Close` as soon as all the records are backfilled. WDYT?

Reply:
can you explain more? we are defering which should take care of all the scenarios
i prefer defer than explicity closing after a certain operation